gitlab/resource_gitlab_project_test.go

```diff
@@ -724,6 +857,58 @@ resource "gitlab_project" "foo" {
 	`, rInt, rInt)
 }
 
+func testAccGitlabProjectConfigTemplateName(rInt int) string {
+	return fmt.Sprintf(`
+resource "gitlab_project" "template-name" {
+  name = "template-name-%d"
+  path = "template-name.%d"
+  description = "Terraform acceptance tests"
+  template_name = "rails"
+  default_branch = "master"
+}
+	`, rInt, rInt)
+}
+
+func testAccGitlabProjectConfigTemplateNameCustom(rInt int) string {
+	return fmt.Sprintf(`
+resource "gitlab_project" "template-name-custom" {
+  name = "template-name-custom-%d"
+  path = "template-name-custom.%d"
+  description = "Terraform acceptance tests"
+  template_name = "myrails"
```

Comment:
Can you add a comment in this test, either here or in the test above, explaining where this template is created?